EIPS/eip-867.md

```diff
@@ -0,0 +1,190 @@
+- **Simple Summary**: A simplified and layman-accessible explanation of the ERP.
+- **Detailed description**: A human-readable description of the proposed corrective actions and the criteria used to determine the proposed actions.  
+- **Justification**: A concise description of why the corrective actions are both reasonable and unlikely to be challenged by any directly affected party.   
+- **Verification script**: A machine-readable script that outputs one State Change Object. The script should clearly implement the selection and action generating logic outlined in the description such that reviewers can independently re-generate an identical State Change Object.  
+- **State Change Object**: The output of the verification script and the input to the ethereum clients.  Primarily, it specifies the complete set of proposed state change actions. 
+- **Appendix (optional)**: with supporting evidence. Attachments in the appendix may include documents verifying details specified as part of the recovery proposal description. 
+
+ The following sections give more detailed descriptions of the expectations for the Justification, Verification Script, and the format of the State Change Object.
+
+### Justification
+A concise description of why this action is both reasonable (cannot be accomplished without an irregular state change) and unlikely to be challenged by a *directly* affected party. 
+
+**Considerable example** (concise, includes supporting evidence, no negative impact): 
+*A crowdsale run by XYZ incorrectly published the testnet address of their crowdsale contract to their public website at the start of their crowdsale on Jan 19, 2018.  501 ETH was sent by 328 users on the mainnet to the incorrect address between block 4,235,987 and 4,236,050.  See here for the testnet contract, and see here for the transactions to the same address on the mainnet.  See here for a statement made by XYZ on their website.  Because there is a contract at this address on the testnet, it is considered effectively impossible that anyone coincidentally holds the private key. We have verified that all transactions came from addresses with no associated code, so there should be no issue returning eth to the senders.*  
+
+**Insufficient example** (not enough detail, no supporting evidence): 
+*We accidentally put the wrong contract address up on our website.  Can you please refund any eth sent to 0x1234 back to the senders.  Thanks.*
+
+**Unacceptable example** (not objective, one person’s word against another):
+*I sent tokens to X for services and he did a lousy job.  I want my money back, but he won’t refund me.  Please help!!*
+
+### Verification Script
+A machine-readable script that outputs a single State Change Object. This script should be implemented so that it is easily audited by a reviewer. Verification scripts should be javascript files that may use the web3.js(https://github.com/ethereum/web3.js/) library. 
```

Comment:
Please use the markdown notation `[web3.js](https://github.com/ethereum/web3.js/)`.